docs/detections/rules-ui-manage.asciidoc

```diff
@@ -152,14 +155,14 @@ image::images/rule-snoozing.png[Rules snooze options,65%]
 [[import-export-rules-ui]]
 === Export and import rules
 
-You can export custom detection rules to an `.ndjson` file, which you can then import into another {elastic-sec} environment. 
-
-[NOTE]
-====
-You cannot export Elastic prebuilt rules, but you can duplicate a prebuilt rule, then export the duplicated rule.
+.Requirements
+[sidebar]
+--
+* To learn which subscription you need for exporting and importing custom rules and prebuilt rules (modified and unmodified), refer to the https://www.elastic.co/subscriptions/cloud[Elastic Cloud] subscription page.
```

Comment:
There is also Elastic Stack subscriptions, should we mention it too? https://www.elastic.co/subscriptions

Reply:
Yeah, good idea to ref the self-managed subscriptions page. Thanks!

```suggestion
* To learn which subscription you need for exporting and importing custom rules and prebuilt rules (modified and unmodified), refer to the subscription page for https://www.elastic.co/subscriptions/cloud[{ecloud}] and {subscriptions}[{stack}/self-managed]
```